app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -11,22 +11,22 @@
     </div>
   </div>
 
-<p>If you’re studying for a degree and interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom. You’ll get to experience a range of activities to help you get a feel for school life.</p>
+<p>If you’re studying for an undergraduate degree or a master's degree and are interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom. You’ll get to experience a range of activities to help you get a feel for school life.</p>
```

Comment:
```suggestion
<p>If you’re studying for an undergraduate or master's degree, a teaching internship could help you understand what it’s really like in the classroom. You’ll get to experience a range of activities and get a feel for school life.</p>
```
first sentence is a bit lengthy right now - worth shortening? and worth removing repetition of 'help'?